src/platforms/java/common/performance/index.mdx

```diff
@@ -166,3 +166,59 @@ if (span != null) {
     }
 }
 ```
+
+### Connecting Errors with Spans
+
+Sentry errors can be linked with transactions and spans.
+
+Errors reported to Sentry while transaction or span is running are linked automatically:
+
+```java {tabTitle: Java}
+ISpan span = Sentry.startTransaction("task");
+try {
+    processItem();
+} catch (Exception e) {
+    Sentry.captureException(e);
+} finally {
+    span.finish();
+}
+```
+
+```kotlin {tabTitle: Kotlin}
+val span = Sentry.startTransaction("task")
+try {
+    processItem()
+} catch (e: Exception) {
+    Sentry.captureException(e)
+} finally {
+    span.finish()
+}
+```
+
+In cases where the decision if thrown exception should be reported to Sentry happens later in the future after span is already finished, to connect error with a span you must link the exception with a Span by calling `setThrowable` method:
```

Comment:
This is confusing. I think this means (and feel free to modify if I've got this wrong):
```suggestion
In future, if a thrown exception should be reported to Sentry after the span has already finished, you must link the exception with the span by calling `setThrowable` method to connect the error:
```

Reply:
I am not sure if "In future" here is understandable for everyone. Maybe we can just leave "If a thrown exception ..." ?